src/gui/transferlistmodel.cpp

```diff
@@ -434,6 +440,10 @@ QString TransferListModel::displayValue(const BitTorrent::Torrent *torrent, cons
         return availabilityString(torrent->distributedCopies());
     case TR_TOTAL_SIZE:
         return unitString(torrent->totalSize());
+    case TR_INFOHASH_V1:
+        return torrent->infoHash().v1().toString();
```

Comment:
I think it should return "N/A" instead of empty string for unavailable info hash.

Reply:
I can do this but columns seem to be blank when there is no value (Category, Tags, Last Seen Complete, Completed On...)

Reply:
Ok, let's keep it as is for now.